pkg/spdk/controllerserver.go

```diff
@@ -263,8 +269,10 @@ func prepareCreateVolumeReq(ctx context.Context, req *csi.CreateVolumeRequest, s
 		MaxRWmBytes: params["qos_rw_mbytes"],
 		MaxRmBytes:  params["qos_r_mbytes"],
 		MaxWmBytes:  params["qos_w_mbytes"],
+		MaxSize:     params["max_size"],
```

Comment:
the type of `params["max_size"]` is a string. Does the API also requires this value to be a string?

Reply:
yes the api converts it back to `int`